packages/driver/src/cypress/log.js

```diff
@@ -7,13 +7,14 @@ const $Events = require('./events')
 const $dom = require('../dom')
 const $utils = require('./utils')
 const $errUtils = require('./error_utils')
+const { getIndexOfUserOptions } = require('../cy/options')
 
 // adds class methods for command, route, and agent logging
 // including the intermediate $Log interface
 const groupsOrTableRe = /^(groups|table)$/
 const parentOrChildRe = /parent|child/
 const SNAPSHOT_PROPS = 'id snapshots $el url coords highlightAttr scrollBy viewportWidth viewportHeight'.split(' ')
-const DISPLAY_PROPS = 'id alias aliasType callCount displayName end err event functionName hookName instrument isStubbed message method name numElements numResponses referencesAlias renderProps state testId timeout type url visible wallClockStartedAt'.split(' ')
+const DISPLAY_PROPS = 'id actual alias aliasType callCount displayName end err event expected functionName hookName instrument isStubbed message method name numElements numResponses referencesAlias renderProps state subject testId type url visible options'.split(' ')
```

Comment:
`timeout` and `wallClockStartedAt` are both required in order for the timeout bar to show up properly.

It looks like our tests don't detect this as they're mostly checking the front-end components. Perhaps we can add a better e2e test that includes this driver code.

In addition, `options` should be placed alphabetically within the list.